example_site/src/components/ConfigurationControls.css

```diff
@@ -22,27 +22,78 @@
 
 button.configuration-submit-button {
   display: block;
-  margin-top: 40px;
+  margin-top: 20px;
   text-align: center;
 }
 
-.configuration-controls-subsection > label,
-.configuration-controls-subsection > input {
-  /* Indent contents of subsection to visually separate them */
-  margin-left: 16px;
+.configuration-controls-subsection {
+  padding: 10px;
 }
 
 /* Align video confirmation modal with unsaved changes modal */
 .modal-align {
-  margin-top: 49px;
+  margin-top: 38px;
+}
+
+.select-input {
+  background-color: #fff;
+  padding: 10px 32px 10px 12px;
+  border-radius: 4px;
+  border: 1px solid #b3bfcc;
+  -webkit-appearance: none;
+  background-image: url(
+    "data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http://www.w3.org/2000/svg' width='14' height='10' viewBox='0 0 17 10'%3E%3Cpath fill='%251d2129' d='M8.405 10h-.007a1.21 1.21 0 01-.865-.375L.351 2.131A1.275 1.275 0 01.365.36a1.203 1.203 0 011.73.015l6.322 6.597 6.5-6.608a1.2 1.2 0 011.729.008 1.275 1.275 0 01-.008 1.771L9.266 9.637a1.207 1.207 0 01-.86.363'/%3E%3C/svg%3E");
+  background-position: calc(100% - 10px) 50%;
+  background-repeat: no-repeat;
+}
+
+.select-input:active,
+.select-input:focus {
+  border-color: #005aff;
+  background-color: #e9f4fc;
+  outline: none;
 }
 
 /* Center form options */
 .center-form {
   margin: auto;
   text-align: left;
   width: 90%;
-  padding: 10px;
+  padding-top: 10px;
+}
+
+.setting-title {
+  font-size: 25px;
+  letter-spacing: -1px;
+  font-weight: 550;
+}
+
+.form-input {
+  line-height: 1.1875;
+  letter-spacing: normal;
+  background-color: rgb(255, 255, 255);
+  padding: 10px 12px;
+  margin: 0px;
+  border-radius: 4px;
+  border: 1px solid rgb(179, 191, 204);
+  transition: all 0.2s ease-in-out 0s;
+  appearance: none;
+  resize: none;
+  overflow: hidden;
+  top: 0px;
+  left: 0px;
+  width: 100%;
+  word-break: break-word;
+  -moz-box-sizing: border-box; 
+  -webkit-box-sizing: border-box; 
+  box-sizing: border-box; 
+}
+
+.form-input:active,
+.form-input:focus {
+  border-color: #005aff;
```

Comment:
Let's start using [CSS variables](https://www.w3schools.com/css/css3_variables.asp)!

I think this will include creating CSS constant files with variables declared in the `:root` selector - let's go over this during our 1:1.